graviti/openapi/policy.py

```diff
@@ -50,75 +53,130 @@ def _request_get_policy(self) -> Dict[str, str]:
             expired=_EXPIRED_IN_SECOND,
         )["policy"]
 
-    def _init_get_policy(self) -> Dict[str, str]:
+    def _request_put_policy(self) -> Dict[str, str]:
+        dataset = self._dataset
+        return get_object_policy(  # type: ignore[no-any-return]
+            dataset.access_key,
+            dataset.url,
+            dataset.owner,
+            dataset.name,
+            actions="PUT",
+            is_internal=config.is_internal,
+            expired=_EXPIRED_IN_SECOND,
+        )["policy"]
```

Comment:
This seems identical to `_request_get_policy`, why not define a `_request_policy` method and pass the action to it?